packages/webos/LS2Request/LS2Request.js

```diff
@@ -126,14 +126,23 @@ export default class LS2Request {
 		this.timeoutJob.stop();
 
 		let parsedMsg;
-		try {
-			parsedMsg = JSON.parse(msg);
-		} catch (e) {
+
+		if (msg == null) {
 			parsedMsg = {
 				errorCode: -1,
-				errorText: msg,
+				errorText: 'No message returned',
```

Comment:
This error message seems a touch weak.  What about 'Invalid response' or something else that is more clear that an error occurred?  I worry that a developer might be confused about whether it was just an empty message vs. a response that's not expected.

Reply:
works for me